### PR TITLE
OP clip, merge_lod_tensor, convert/elementwise error message enhancement

### PR DESCRIPTION
**Pull request by haozech**
test=develop

### CODE REVIEW

File: paddle/fluid/inference/tensorrt/convert/elementwise_op.cc [hunk 1]
```diff
@@ -152,7 +168,10 @@ class ElementwiseTensorOpConverter : public OpConverter {
   void operator()(const framework::proto::OpDesc& op,
                   const framework::Scope& scope, bool test_mode) override {
     auto op_pair = ops.find(op_type_);
-    PADDLE_ENFORCE(op_pair != ops.end(), "Wrong elementwise op type!");
+    PADDLE_ENFORCE_NE(op_pair, ops.end(),
+                      platform::errors::InvalidArgument(
+                          "Elementwise op's type is not supported. Please "
+                          "check if the op_type is correct."));
```
Comment (NHZlX) [inline paddle/fluid/inference/tensorrt/convert/elementwise_op.cc]: 打印下具体的type

Comment (haozech) [reply paddle/fluid/inference/tensorrt/convert/elementwise_op.cc]: 已修改 感谢

**Review by NHZlX — DISMISSED** [review]
LGTM to the eltwise convert err message.

File: paddle/fluid/inference/tensorrt/convert/elementwise_op.cc [hunk 2]
```diff
@@ -99,8 +99,10 @@ class ElementwiseWeightOpConverter : public OpConverter {
         regist_eltwise_weight(scale_mode);
       } else {
         PADDLE_THROW(platform::errors::InvalidArgument(
-            "TensorRT Dynamic shape unsupported weight shape for Elementwise "
-            "op!"));
+            "The size of input bias's dims is %d, but TensorRT dynamic shape "
+            "only support"
```
Comment (fc500110) [inline paddle/fluid/inference/tensorrt/convert/elementwise_op.cc]: 空格

File: paddle/fluid/inference/tensorrt/convert/elementwise_op.cc [hunk 3]
```diff
@@ -132,12 +134,26 @@ class ElementwiseWeightOpConverter : public OpConverter {
       if (scale_mode == nvinfer1::ScaleMode::kCHANNEL) {
         for (size_t i = 1; i < no_batch_dims.size(); i++) {
           if (dims_y[i] != 1)
-            PADDLE_THROW(
-                "TensorRT unsupported weight shape for Elementwise op!");
+            PADDLE_THROW(platform::errors::InvalidArgument(
+                "The bias's %d dim is %d, but TensorRT dynamic shape only "
+                "support"
```
Comment (fc500110) [inline paddle/fluid/inference/tensorrt/convert/elementwise_op.cc]: "support "

**Review by fc500110 — CHANGES_REQUESTED** [review]

File: paddle/fluid/inference/tensorrt/convert/elementwise_op.cc (hunk 2 above)
Comment (haozech) [reply paddle/fluid/inference/tensorrt/convert/elementwise_op.cc]: 已修改 感谢

File: paddle/fluid/inference/tensorrt/convert/elementwise_op.cc (hunk 3 above)
Comment (haozech) [reply paddle/fluid/inference/tensorrt/convert/elementwise_op.cc]: Done. THX

**Review by fc500110 — APPROVED** [review]
LGTM

**Pull request merged**